sql/core/benchmarks/ExtractBenchmark-results.txt

```diff
@@ -1,119 +1,119 @@
+MINUTE of date                                     1338           1342           4          7.5         133.8       0.4X
+SECOND of date                                     1483           1498          20          6.7         148.3       0.4X
+MILLISECONDS of date                               1473           1503          28          6.8         147.3       0.4X
+MICROSECONDS of date                               1379           1393          22          7.3         137.9       0.4X
+EPOCH of date                                      1510           1546          60          6.6         151.0       0.4X
 
-OpenJDK 64-Bit Server VM 1.8.0_232-8u232-b09-0ubuntu1~18.04.1-b09 on Linux 4.15.0-1044-aws
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_231-b11 on Mac OS X 10.15.3
+Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
 Invoke date_part for interval:            Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-cast to interval                                   1249           1254           6          8.0         124.9       1.0X
-MILLENNIUM of interval                             1310           1316           9          7.6         131.0       1.0X
-CENTURY of interval                                1304           1315          10          7.7         130.4       1.0X
-DECADE of interval                                 1306           1313           7          7.7         130.6       1.0X
-YEAR of interval                                   1304           1313          11          7.7         130.4       1.0X
-QUARTER of interval                                1310           1317           7          7.6         131.0       1.0X
-MONTH of interval                                  1311           1319          12          7.6         131.1       1.0X
-DAY of interval                                    1295           1304          13          7.7         129.5       1.0X
-HOUR of interval                                   1301           1306           8          7.7         130.1       1.0X
-MINUTE of interval                                 1316           1319           3          7.6         131.6       0.9X
-SECOND of interval                                 1437           1440           3          7.0         143.7       0.9X
-MILLISECONDS of interval                           1435           1449          16          7.0         143.5       0.9X
-MICROSECONDS of interval                           1304           1314           9          7.7         130.4       1.0X
-EPOCH of interval                                  1440           1453          19          6.9         144.0       0.9X
+cast to interval                                   3984           4034          79          2.5         398.4       1.0X
```

Comment:
So, this slowdown came from the change, right?

Reply:
Hold on a bit... something wrong here

Reply:
I fixed the problem, thanks, @dongjoon-hyun.

the benchmark `cast to interval` is slower because of the extra cast to string logic.

Reply:
the extract-related benchmark tests are ok now